### ISSUE TITLE
Next Stable Version Release?

### ISSUE THREAD

**Issue by Arkanyus** [greenshot/greenshot#492]
When is the next stable version release planned for? / What is the blocker?

Last stable release was 6 years ago, and I can see from the version history that it appeared to be that a stable release was gearing up last year with the monthly unstable releases.

Difficult to justify this as best practice usable when it doesn't *appear* to be supported.

As i can see plenty of activity on this here GitHub.

Kind Regards,

**Comment by FF-Brown** [comment]
Last I heard there was an issue with certs for the installer. Maybe just with funding them? 
If that's the case, @Lakritzator, how much is it for the cert?

**Comment by Arkanyus** [comment]
Thanks for the response - don't suppose you've heard anything back on this?

**Comment by FF-Brown** [comment]
Nope, nothing yet. I'd love to help the project move forward but communication has been a bit slow (understandable considering this is a side project for any of us who work on it).

**Comment by mariusi7** [comment]
Only today when I had to "fix" the issue with Windows stealing back PrintScr key I realized how **old** this is. Could have a bunch of vulnerabilities, no ? If no new release soon, anyone knows an updated alternative just as good ?

**Comment by eroldan** [comment]
> Only today when I had to "fix" the issue with Windows stealing back PrintScr key I realized how **old** this is. Could have a bunch of vulnerabilities, no ? If no new release soon, anyone knows an updated alternative just as good ?

I moved to Flameshot and is pretty good, maintained open source.

**Comment by ben-ekw** [comment]
FYI, GreenShot seems to be broken on the latest Windows 11 Update

**Comment by bakx** [comment]
> Flameshot

Took a look at it, but doesn't seem to support region capture (or I didn't see it in the short cuts)..   I applied the FYI above to fix Greenshot capture and I'll stick with Greenshot until Flameshot adds region captures.

**Comment by koppor** [comment]
- Code signing is https://github.com/greenshot/greenshot/issues/264
- Please think of putting 10$ to https://opencollective.com/greenshot (if all readers to that, the new laptop for development would be there)

---

@bakx - Please explain what you mean by "region capture"?

![grafik](https://github.com/greenshot/greenshot/assets/1366654/5fdb30b6-ffd3-40a8-88ac-3c4e11506d67)

Flameshot is IMHO unable to 1) craft rectrangles with transparancy and b) offering the editor open AND to have the screenshot in the clipboard. this is is a productivity killer feature!

See https://alternativeto.net/software/greenshot/about/ for a full list of alternatives to Greenshot. I tried many of them and turned back to Greenshot.

**Comment by jklingen** [comment]
Hi folks,
sorry, everything's going slowly for quite some time now 😕  We just posted a new blog post about the [current status of Greenshot](https://getgreenshot.org/2024/02/11/current-status-greenshot/).
I hope it adds some clarity.

> tl;dr: We are struggling with a lack of time, but are gradually trying to pick up speed again.

**Comment by Lakritzator** [comment]
Sorry for the LONG wait, but my lovely colleagues have finally managed that what I didn't have the time for: https://getgreenshot.org/2025/07/14/final-release-greenshot-1-3/
Closing this issue
